dm/ctl/master/operate_source.go

```diff
@@ -48,31 +48,36 @@ func convertCmdType(t string) pb.SourceOp {
 
 // operateMysqlFunc does migrate relay request
 func operateSourceFunc(cmd *cobra.Command, _ []string) {
-	if len(cmd.Flags().Args()) != 2 {
+	if len(cmd.Flags().Args()) < 2 {
 		cmd.SetOut(os.Stdout)
 		cmd.Usage()
 		return
 	}
 
 	cmdType := cmd.Flags().Arg(0)
-	configFile := cmd.Flags().Arg(1)
-	content, err := common.GetFileContent(configFile)
-	if err != nil {
-		common.PrintLines("get file content error:\n%v", err)
-		return
-	}
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-
 	op := convertCmdType(cmdType)
 	if op == pb.SourceOp_InvalidSourceOp {
 		common.PrintLines("invalid operate '%s' on worker", cmdType)
 		return
 	}
 
+	contents := make([]string, len(cmd.Flags().Args())-1)
+	for i := 1; i < len(cmd.Flags().Args()); i++ {
+		configFile := cmd.Flags().Arg(i)
+		content, err := common.GetFileContent(configFile)
+		if err != nil {
+			common.PrintLines("get file content error:\n%v", err)
```

Comment:
Should we also print filename which is failed?

Reply:
original `err` should already contain filename